packages/graph/hash_graph/bench/benches/representative_read/mod.rs

```diff
@@ -0,0 +1,115 @@
+mod knowledge;
+mod ontology;
+
+use std::{collections::HashMap, str::FromStr};
+
+use criterion::{BenchmarkId, Criterion};
+use criterion_macro::criterion;
+use graph::{
+    knowledge::{Entity, EntityId},
+    ontology::AccountId,
+    store::{AccountStore, EntityStore},
+};
+use graph_test_data::{data_type, entity, entity_type, link_type, property_type};
+use rand::{seq::IteratorRandom, thread_rng};
+use type_system::uri::{BaseUri, VersionedUri};
+use uuid::Uuid;
+
+use crate::util::{setup, StoreWrapper};
+
+/// DOC - TODO
```

Comment:
Not sure this is worth an Asana task, it can be address as the `Samples` object is fleshed out and the shape of it is better defined